chat-api/v2/openapi.yaml

```diff
@@ -210,6 +215,22 @@ paths:
                 status: 404
         default:
           $ref: '#/components/responses/Default'
+  /channels/whatsapp/messages/{message-id}:
+    parameters:
+      - $ref: '#/components/parameters/MoMessageId'
+    put:
+      operationId: markWhatsAppMessageAsRead
+      summary: Mark received whatsapp message as read
```

Comment:
I guess this will be the TOC displayed title. I guess it is too long and it will be not easily readable. Not critical but if you can find something shorter would be nice

Reply:
shortened